File: condicionales if else.php
<?php
// Parte 1: Condición simple
$a = 2;
if ($a == 2) {
    echo "a vale 2\n";
} else {
    echo "a es diferente de 2\n";
}

// Parte 2: Concatenar condiciones
$Nombre = 'Santiago';
$Edad = 23;
$Pais = 'Colombia';

if ($Nombre == 'Santiago' && $Edad == 23 && $Pais == 'Colombia') {
    echo "Su nombre es $Nombre, tiene $Edad y es de $Pais\n";
} elseif ($Nombre == 'Santiago' && !($Edad == 23)) {
    echo "Su nombre es Santiago y no tiene 23 años\n";
} elseif ($Nombre != 'Santiago' && $Edad == 23) {
    echo "Su nombre no es Santiago y tiene 23 años\n";
} else {
    echo "No se llama Santiago ni tiene 23 años\n";
}
?>

File: Condicionales If Else.py
a=2
if a ==2:
    print('a vale 2')
else:
    print('a es diferente de 2')

#Concatenar condiciones 

Nombre = 'Santiago'
Edad = 23
Pais='Colombia'

if Nombre =='Santiago' and Edad == 23 and Pais =='Colombia':
    print('Su nombre es', Nombre,'tiene',Edad,'y es de',Pais) 
elif Nombre == 'Santiago'and not Edad==23:
    print('Su nombre es Santiago y no tiene 23 años')
elif Nombre != 'Santiago'and Edad == 23:
    print('Su nombre no es Santiago y tiene 23 años')
else:
    print('No se llama Santiago ni tiene 23 años')         